examples/app/main.py

```diff
@@ -25,7 +27,8 @@ class ExampleApp(LabServerApp):
     extension_url = "/lab"
     default_url = "/lab"
     name = __name__
-    load_other_extensions = False
+    # In jupyter-server v2 terminals are an extension
+    load_other_extensions = True
```

Comment:
It would be nice to keep that one to `False` as one of the main purposes of these examples is to show how to create *standalone* applications.

If that makes the tests fail maybe the test should be updated to not expect terminals anymore?

Reply:
The tests check that JupyterLab starts without throwing an error in the dev console.
We could make the terminal service optional or catch the error and show a warning.

But I'm not sure what is the best solution.

Reply:
I'm looking into it.